<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAudioSetupComponent</class>
 <widget class="QFrame" name="CAudioSetupComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio setup</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="vl_AudioSetup">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QFormLayout" name="fl_SetupAudio">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_SetupAudioInputDevice">
       <property name="text">
        <string>In</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_SetupAudioInputDevice">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_SetupAudioOutputDevice">
       <property name="text">
        <string>Out</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_SetupAudioOutputDevice">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_SetupAudioLoopback">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="cb_SetupAudioLoopback">
       <property name="text">
        <string>Loopback, test sound in- to output</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lbl_ExtraInfo">
       <property name="text">
        <string>Extra info goes here</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gl_NotificationSounds">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_NotificationSounds">
       <property name="text">
        <string>Notification sounds</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="tb_ExpandNotificationSounds">
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Toggle visibility</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../blackmisc/blackmisc.qrc">
         <normaloff>:/diagona/icons/diagona/icons/toggle-expand.png</normaloff>
         <normalon>:/diagona/icons/diagona/icons/toggle.png</normalon>:/diagona/icons/diagona/icons/toggle-expand.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QFrame" name="fr_NotificationSoundsInner">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="cb_SetupAudioNotificationVoiceRoom">
          <property name="text">
           <string>notification for join/leave voice room</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_SetupAudioPlayNotificationSounds">
          <property name="text">
           <string>play notification sounds</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_SetupAudioNotificationTextMessage">
          <property name="text">
           <string>notification for text messages</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vs_AudioSetup">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>143</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
